apps/portal/src/pages/index.astro

```diff
@@ -0,0 +1,56 @@
+---
+const isDev = import.meta.env.DEV;
+
+const title = isDev ? 'iTwinUI (local)' : 'iTwinUI';
+const faviconHref = isDev ? 'favicon-dev.svg' : 'favicon.svg';
+
+const links = [
+  { name: 'html pages', href: isDev ? 'http://localhost:5173' : 'css' },
```

Comment:
We could add link to Layouts demo page to have all links in one place.

Reply:
can we do that in a future PR? this is blocking #937 which is important for users

Reply:
also layouts doesn't exist in this repo, so we dont know about its dev server

Reply:
Agree with future PR. 
It doesn't matter that layouts does not exist in this repo. You can add github pages link to both cases. It's just a better developer/user experience to have all packages demo pages in one place if this portal will be deployed as a front page for all devs.

Reply:
Same with icons.